python/lsst/eotest/image_utils.py

```diff
@@ -389,8 +400,10 @@ def superbias(files, overscan, imaging=None, dxmin=5, dxmax=2, bias_method='row'
     """Generates a single stacked 'super' bias frame based on
     a statistic. Images must be either all masked or all unmasked."""
     ims = [afwImage.ImageF(f, hdu) for f in files]
-    bias_frames = [unbias_and_trim(im, overscan, imaging, dxmin, dxmax, bias_method,
-                                   **kwargs) for im in ims]
+    bias_frames = []
```

Comment:
There's no actual change in functionality here.

Reply:
Yes, you are right, I had made that change while debugging the code.  I went ahead and put it back as it was.